Fixes problem with setting the TLS client protocol version and ciphersuite
in the NSSWITCH LDAP library in Solaris.
Patch was developed in-house; it is Solaris specific and
will not be contributed upstream.

diff --git a/libraries/libldap/ldap.conf b/libraries/libldap/ldap.conf
index a94cfaa..e48757b 100644
--- a/libraries/libldap/ldap.conf
+++ b/libraries/libldap/ldap.conf
@@ -11,3 +11,6 @@
 #SIZELIMIT	12
 #TIMELIMIT	15
 #DEREF		never
+
+TLS_PROTOCOL_MIN	3.2
+TLS_CIPHER_SUITE	TLSv1.2:!aNULL:!eNULL:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA:DHE-RSA-AES256-SHA:DHE-DSS-AES256-SHA:DHE-RSA-DES-CBC3-SHA:DHE-DSS-DES-CBC3-SHA:AES128-SHA:AES256-SHA:DES-CBC3-SHA
diff --git a/servers/slapd/slapd.conf b/servers/slapd/slapd.conf
index b225fe5..06e50f0 100644
--- a/servers/slapd/slapd.conf
+++ b/servers/slapd/slapd.conf
@@ -23,6 +23,8 @@ argsfile	%LOCALSTATEDIR%/run/slapd.args
 #	Require 112-bit (3DES or better) encryption for updates
 #	Require 63-bit encryption for simple bind
 # security ssf=1 update_ssf=112 simple_bind=64
+TLSProtocolMin	3.2
+TLSCipherSuite	TLSv1.2:!aNULL:!eNULL:DHE-RSA-AES128-SHA:DHE-DSS-AES128-SHA:DHE-RSA-AES256-SHA:DHE-DSS-AES256-SHA:DHE-RSA-DES-CBC3-SHA:DHE-DSS-DES-CBC3-SHA:AES128-SHA:AES256-SHA:DES-CBC3-SHA
 
 # Sample access control policy:
 #	Root DSE: allow anyone to read it
